sentry_youtrack/forms.py

```diff
@@ -132,6 +132,7 @@ class YouTrackConfigurationForm(forms.Form):
         'client': _("Unable to connect to YouTrack."),
         'invalid_ssl': _("SSL certificate  verification failed."),
         'invalid_password': _('Invalid username or password.'),
+        'invalid_project': _('Invalid project: \'%s\''),
```

Comment:
No.
`client` message is used when we cannot connect to YouTrack.
Here we can but cannot get any project as the URL is wrong.